docs/cli.md

```diff
@@ -183,7 +183,9 @@ We do not collect private information such as IP addresses, contents of ingestio
 The code responsible for collecting and broadcasting these events is open-source and can be found [within our GitHub](https://github.com/linkedin/datahub/blob/master/metadata-ingestion/src/datahub/telemetry/telemetry.py).
 
 Telemetry is enabled by default, and the `telemetry` command lets you toggle the sending of these statistics via `telemetry enable/disable`.
-You can also disable telemetry by setting `DATAHUB_TELEMETRY_ENABLED` to `false`.
+You can also disable telemetry by setting `DATAHUB_TELEMETRY_ENABLED` to `false`. If you are running CLI in a private environment with no access to public internet then you need to disable telemetry.
+
+You can set `DATAHUB_TELEMETRY_TIMEOUT` to an integer value to specify timeout in secs when sending telemetry.
```

Comment:
Please clarify that these are OS environment variables. Callout the default value of `DATAHUB_TELEMETRY_TIMEOUT` as 10 seconds.